src/toil/wdl/wdltoil.py

```diff
@@ -772,6 +807,115 @@ def is_url(filename: str, schemes: List[str] = ['http:', 'https:', 's3:', 'gs:',
                 return True
         return False
 
+def convert_remote_files(environment: WDLBindings, file_source: Toil, task_path: str, search_paths: Optional[List[str]] = None, skip_remote: bool = False) -> None:
+    """
+    Iterate through the environment and convert all files that reference a remote file from the possible URIs to that URI.
+    If the file references a local file, leave it unchanged.
+    :param environment: Bindings to evaluate on
+    :param file_source: Context to search for remote files with
+    :param task_path: Dotted WDL name of the user-level code doing the
+           importing (probably the workflow name).
+    :param search_paths: If set, try resolving input location relative to the URLs or
+           directories in this list.
+    :param skip_remote: If set, don't try to import files from remote
+           locations. Leave them as URIs.
```

Comment:
I changed the argument to better reflect what it does. I called it convert since it not only downloads/imports the files, but sets the value in the WDL file value to the new URI.